test.dockerfile

```diff
@@ -35,7 +35,8 @@ RUN pip install tox
 # Install the relevant dependencies. Keep printing so Travis knows we're alive.
 RUN ["bash", "-c", "( while true; do echo '.'; sleep 60; done ) & tox --notest"]
 
-# Finally, upload our actual code!
+# Finally, cleans cached code and upload our actual code!
+RUN rm -rf ./gym*
```

Comment:
`rm -rf .` doesn't work, because you can't remove the current directory, and we would need to recreate the directory to move .tox back into it. Replaced ./.tox with .tox, and $HOME with /tmp